specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-03-01-preview/loadtestservice.json

```diff
@@ -0,0 +1,2593 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Load Testing",
+    "version": "2022-03-01-preview"
+  },
+  "paths": {
+    "/appcomponents/{name}": {
+      "patch": {
+        "tags": [
+          "AppComponent"
+        ],
+        "summary": "Associate app component (a azure resource model) to a test model or test run",
+        "operationId": "AppComponent_CreateOrUpdateAppComponents",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "name",
+            "description": "Unique identifier for app component name, must be a valid URL character ^[a-z0-9_-]*$",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "Api version",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "App component model",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AppComponentsMap"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The app component were updated",
+            "schema": {
+              "$ref": "#/definitions/AppComponentsMap"
+            }
+          },
+          "201": {
+            "description": "The app component were created",
+            "schema": {
+              "$ref": "#/definitions/AppComponentsMap"
+            }
+          },
+          "default": {
+            "description": "***Error Responses: ***\n\n * \"400\", \"InvalidHostOrResourceName\", \"Host name or domain is not valid\" .\n\n * \"400\", \"RequiredAppComponentName\", \"name field is required\"\n\n * \"400\", \"RequiredTestOrTestRun\", \"Either test name or test run name is required\"\n\n * \"400\", \"InvalidAppComponentName\", \"name field should be valid\"\n\n * \"400\", \"InvalidValue\",  \"value field is either null or not a valid map\".\n\n * \"404\", \"TestRunNotFound\", \"test run not found with given name\"\n\n * \"404\", \"TestNotFound\", \"Test not found with given name\"\n\n* \"500\", \"InternalServerError\", \"Unable to process the request, please retry\"",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
```

Comment:
@mikekistler  can we remove all the error responses from the description in the new version, and keep a default error response, as it will be hard to maintain and mentioned all the error responses in the future. We haven't published our public API docs for the data plane yet so can we do these changes?

Reply:
Yes you can remove the details of the specific errors from the description.